tests/api_tests/api_moderator_test.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { request, test } from "@playwright/test";
+import { apiUrl } from "../../utils/apiUrl";
+import { Api } from "../../pages/Api";
+import { apiDataSet } from "../../utils/dataSet";
+
+let guestUser, admin, createdNewAdmin, newAdmin, user, referralUser, pendingAvatarId, newStream, streamAction, currentStatus
```

Comment:
@YulianH pendingAvatarId should be a const, you use it only in one test
createdNewAdmin should be a const, used only in precondition
guestUser const > precondition only
pendingAvatarId  const > one test only
streamAction  const > one test only
currentStatus const > one test only

Reply:
done